src/commands/npm/lerna/release.ts

```diff
@@ -139,6 +128,11 @@ export default class Release extends SfdxCommand {
       lernaRepo.verifySignature(this.flags.sign);
     }
 
+    if (!this.flags.dryrun) {
```

Comment:
I think I originally put this before the publish because there were times that the `git push` would fail and I didn't want to publish in that scenario... But I think having it after the publish makes more sense because it's a bummer to push the changes for a release that never succeeded.

Either way, I think the `git push` should happen _before_ the `waiting for availability`. That step has been known to timeout so it'd be a bummer for the changes to get lost when that happens

Reply:
good call, have seen the same.